### ISSUE TITLE
Enhancement proposal shadow mapping

### ISSUE THREAD

**Issue by b2renger** [cansik/processing-postfx#29]
@cansik : It could be pretty neat to have some shadows available : 
https://forum.processing.org/two/discussion/12775/simple-shadow-mapping

:)

**Comment by cansik** [comment]
Yes that sounds very interesting. The problem is that the current library is only using the 2d color information of the pixels, not the z-buffer. I have to check how we could implement this.
